config/cesm/machines/config_machines.xml

```diff
@@ -2338,6 +2338,89 @@ This allows using a different mpirun command to launch unit tests
+    <MAX_MPITASKS_PER_NODE>36</MAX_MPITASKS_PER_NODE>
+    <PROJECT_REQUIRED>FALSE</PROJECT_REQUIRED>
+    <mpirun mpilib="default">
+      <executable> mpirun </executable>
+    </mpirun>
+    <module_system type="module">
+      <init_path lang="perl">/usr/share/Modules/init/perl.pm</init_path>
+      <init_path lang="python">/usr/share/Modules/init/python.py</init_path>
+      <init_path lang="csh">/usr/share/Modules/init/csh</init_path>
+      <init_path lang="sh">/usr/share/Modules/init/sh</init_path>
+      <cmd_path lang="perl">/usr/bin/modulecmd perl</cmd_path>
+      <cmd_path lang="python">/usr/bin/modulecmd python </cmd_path>
+      <cmd_path lang="sh">module</cmd_path>
+      <cmd_path lang="csh">module</cmd_path>
+      <modules>
+        <command name="purge"/>
+      </modules>
+      <modules compiler="intel19">
+        <command name="load">intel19.5/19.5.281</command>
+        <command name="load">intel19.5/szip/2.1.1</command>
+        <command name="load">cmake/3.15.0</command>
+        <command name="load">curl/7.66.0</command>
+        <command name="load">anaconda/2.7</command>
+      </modules>
+      <modules mpilib="mpi-serial">
+        <command name="load">intel19.5/hdf5/1.10.5</command>
+        <command name="load">intel19.5/netcdf/C_4.7.2-F_4.5.2_CXX_4.3.1</command>
+      </modules>
+      <modules mpilib="!mpi-serial">
+	<command name="load">impi19.5/19.5.281</command>
+        <command name="load">impi19.5/hdf5/1.10.5</command>
+        <command name="load">impi19.5/netcdf/C_4.7.2-F_4.5.2_CXX_4.3.1</command>
+        <command name="load">impi19.5/parallel-netcdf/1.12.0</command>
+      </modules>
+      <modules compiler="intel19" mpilib="!mpi-serial" DEBUG="TRUE">
+        <command name="load">impi19.5/esmf/8.0.0-intelmpi-64-g</command>
+      </modules>
+      <modules compiler="intel19" mpilib="!mpi-serial" DEBUG="FALSE">
+        <command name="load">impi19.5/esmf/8.0.0-intelmpi-64-O</command>
+      </modules>
+      <modules compiler="intel19" mpilib="mpi-serial" DEBUG="TRUE">
+        <command name="load">intel19.5/esmf/8.0.0-mpiuni-64-g</command>
+      </modules>
+      <modules compiler="intel19" mpilib="mpi-serial" DEBUG="FALSE">
+        <command name="load">intel19.5/esmf/8.0.0-mpiuni-64-O</command>
+      </modules>
+    </module_system>
+    <environment_variables>
+      <env name="OMP_STACKSIZE">256M</env>
+    </environment_variables>
+    <environment_variables compiler="intel19">
+      <env name="I_MPI_EXTRA_FILESYSTEM">1</env>
+      <env name="I_MPI_EXTRA_FILESYSTEM_FORCE">gpfs</env>
+      <env name="I_MPI_DEBUG">60</env>
+      <env name="GPFSMPIO_TUNEBLOCKING">0</env>
+      <env name="I_MPI_PLATFORM">skx</env>
+      <env name="I_MPI_SHM">skx_avx512</env>
+      <env name="I_MPI_HYDRA_BOOTSTRAP">lsf</env>
+      <env name="I_MPI_LSF_USE_COLLECTIVE_LAUNCH">1</env>
+      <env name="XIOS_PATH">/work/csp/cmip01/csm/xios</env>
+      <env name="NETCDF_PATH">$ENV{NETCDF}</env>
```

Comment:
You shouldn't need to define NETCDF_PATH and PNETCDF_PATH in both config_compilers.xml and here.

Reply:
NETCDF_PATH and PNETCDF_PATH were put here for a test. I removed them.